jupyterhub/templates/image-puller/_daemonset-helper.yaml

```diff
@@ -82,6 +87,11 @@ spec:
         {{- end }}
         {{- if not .Values.singleuser.cloudMetadata.enabled }}
         - name: image-pull-metadata-block
+          {{- if .Values.hookimagepuller.securityContext.enabled }}
```

Comment:
Can we just let the user specify the whole securityContext? That way, they can set other stuff too (like capabilities, gid, etc)

Reply:
It also means we don't have to use the hub uid everywhere.